src/http.ts

```diff
@@ -284,6 +289,7 @@ export async function extendBundle(
     headers: {
       ...prepareTokenHeaders(options.sessionToken),
       source: options.source,
+      ...(options.requestId && { 'snyk-request-id': options.requestId })
```

Comment:
also, by making it mandatory and having a default we wouldn't have to do the check here.

Reply:
Regardless, we dont have to make a check. What I had to choose was:

A - `snyk-request-id: null`
B - don't send `'snyk-request-id'` in the first place.

I went for B but I wasn't really convinced of the decision

Reply:
I guess if header is `null` it's not being sent at all by design? I can't confirm this, just a guess.

Reply:
you're right, `null` might be an option. 

[seems](https://github.com/sindresorhus/got/issues/1771#issuecomment-871445122) like `node` doesn't like `undefined` headers too. So it's probably right to double check in this case.